executor/load_data.go

```diff
@@ -750,7 +750,20 @@ func (w *fieldWriter) GetField() (bool, field) {
 			flag, ch = w.getChar()
 			if flag {
 				w.OutputBuf = append(w.OutputBuf, '\\')
-				w.OutputBuf = append(w.OutputBuf, ch)
+				if ch == w.enclosedChar {
```

Comment:
It's not a correct fix. The `load data` statement in issue uses `ESCAPED BY ''`. It means the statement doesn't do any escape. The correct fix is to fully support `ESCAPED BY`.

Reply:
Could you test some cases without `ESCAPED BY ''`?

Reply:
In this case without `ESCAPED BY ''` the result is same
```
mysql> load data local infile '/data/load_json.csv' ignore  into table load_json columns terminated by ',' enclosed by '"'  ignore 1 lines (`aaa`);
Query OK, 3 rows affected (0.00 sec)
Records: 3  Deleted: 0  Skipped: 0  Warnings: 0

mysql> select * from load_json; 
+---------------------------------------------------------------------------------------------------------------------+
| aaa                                                                                                                 |
+---------------------------------------------------------------------------------------------------------------------+
| {"itemRangeType":0,"itemContainType":0,"shopRangeType":0}                                                           |
| {"itemRangeType":0,"itemContainType":0,"shopRangeType":1,"shopJson":"[{\"id\":\"A1234\",\"shopName\":\"AAAAAA\"}]"} |
| {"itemRangeType":0,"itemContainType":0,"shopRangeType":1,"shopJson":"[{\"id\":\"A1234\",\"shopName\":\"AAAAAA\"}]"} |
+---------------------------------------------------------------------------------------------------------------------+
3 rows in set (0.00 sec)
```